todo_app/app.py

```diff
@@ -19,10 +22,18 @@ def create_app():
     client_secret = os.getenv("Git_Client_secret")
     client_id = os.getenv('Git_Client_ID')
     client = WebApplicationClient(client_id)
+    
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+    app.logger.info("Value of logger is info")
+    
+
+    if app.config['LOGGLY_TOKEN'] is not None:
+      handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app')
+      handler.setFormatter( Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s") )
+    app.logger.addHandler(handler)
```

Comment:
This needs to be indented so that it's inside the `if` block. You can see [your pipeline is failing](https://github.com/Arpitha-debug/DevOps-Course-Starter/runs/5815282038?check_suite_focus=true) because of this.

- The `if` condition is false during the test run (this is correct)
- So no `handler` variable is created
- So `addHandler(handler)` doesn't work.